packages/sdks-tests/src/e2e-tests/dynamic-unknown-element.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { expect } from '@playwright/test';
+import { test } from '../helpers/index.js';
+
+test.describe('Unknown Element', () => {
+  test('unknown element is rendered', async ({ page, packageName }) => {
+    test.skip(packageName === 'react-native');
```

Comment:
```suggestion
  test('unknown element is rendered', async ({ page, sdk }) => {
    test.skip(sdk === 'react-native');
```

https://github.com/BuilderIO/builder/blob/1396fb4cc5ff9db2369f0cff12f42fbb910071c9/packages/sdks-tests/src/helpers/sdk.ts#L2-L40

`sdk` is `react-native`

`packageName` is `react-native-74` or `react-native-76-fabric` (the specific server that is running the SDK)